src/ds/logger.h

```diff
@@ -38,67 +29,115 @@ namespace logger
 
   static constexpr Level MOST_VERBOSE = static_cast<Level>(0);
 
-  static constexpr long int ns_per_s = 1'000'000'000;
+  static constexpr const char* LevelNames[] = {
+#ifdef VERBOSE_LOGGING
+    "trace",
+    "debug",
+#endif
+    "info",
+    "fail",
+    "fatal"};
```

Comment:
Probably not in the scope of this PR but mentioning it here as related. For releases, this enum only contains 3 values (info, fail and fatal), which is as expected for the enclave but a little too strict on the host as the log level isn't as sensitive and can be set at launch (see https://microsoft.github.io/CCF/main/operations/configuration.html#logging). For example, setting the host log level to "debug" is currently not possible for CCF releases. 

I had a quick go at trying to define different enums for the host and the enclave but it got messy quickly. Would the changes in this PR make this change easier?

Reply:
I think we could achieve that just by changing the guards on `trace` and `debug` from `#ifdef VERBOSE_LOGGING` to `#if defined(VERBOSE_LOGGING) or not defined(INSIDE_ENCLAVE)`, so they're always available on the host. But is there much value in this? Do we have debug logging on the host that we'd like to make available in a release? As I write this, I think "probably", so I'll see if it really is that esy.

Reply:
Yeah this should be possible, modulo some adjustments to make sure the enum values match up between host and enclave, but it's failing without a clear error so I'm going to skip it for now.